src/playground/auth/api-calls/api-calls.component.ts

```diff
@@ -0,0 +1,148 @@
+/**
+ * @license
+ * Copyright Akveo. All Rights Reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import {Component, Inject} from '@angular/core';
+import { NbAuthResult, NbAuthService, NbAuthToken } from '@nebular/auth';
+import {HttpClient, HttpErrorResponse} from '@angular/common/http';
+import {catchError, map} from 'rxjs/operators';
+import { Wine } from './wine';
+import { Router } from '@angular/router';
+import { getDeepFromObject } from '@nebular/auth/helpers';
+import { NB_AUTH_OPTIONS } from '@nebular/auth';
+
+@Component({
+  selector: 'nb-playground-api-calls',
+  styles: [`
+    .rTable {
```

Comment:
what is `rTable`? is this some kind of 3rd party styles? naming convention looks strange